tableau_exporter/tableau_status_exporter.py

```diff
@@ -12,15 +12,6 @@
 
 logger = logging.getLogger('Tableau exporter')
```

Comment:
Minor suggestion: logger name should not contain space. It is identifier hierarchy with `.` like package name.